packages/core/admin/admin/src/content-manager/pages/ListSettingsView/reducer.js

```diff
@@ -68,6 +68,8 @@ const reducer = (state = initialState, action) =>
         break;
       }
       case 'SUBMIT_FIELD_FORM': {
+        draftState.fieldForm = {};
+        draftState.fieldToEdit = '';
```

Comment:
I noticed that there is already an action type 'UNSET_FIELD_TO_EDIT' that does exactly this.

I think the same result could be achieved by dispatching that action after this one. There is also already a handler for closing the modal that dispatches this action, so you could just call it.

https://github.com/strapi/strapi/blob/0fbd3d3b0c48fac09c1d93d351154e482f0c99e9/packages/core/admin/admin/src/content-manager/pages/ListSettingsView/index.js#L119-L130

becomes:
```js
 const handleCloseModal = () => { 
   dispatch({ 
     type: 'UNSET_FIELD_TO_EDIT', 
   }); 
 }; 
  
 const handleSubmitFieldEdit = (e) => { 
   e.preventDefault(); 
   dispatch({ 
     type: 'SUBMIT_FIELD_FORM', 
   }); 
	 handleCloseModal()
 }; 
```

That way this state reset remains defined in a single place. Wdyt?

Reply:
Personally, I think the change should be handled from the reducer but I'm all in for consistency.
Code has been updated. 
Thanks for taking the time to review this PR.

Reply:
I agree that consistency is another good reason to do it this way. I've approved and will merge as is. I would still be interested in your reasoning for why this should be handled in the reducer though, if you care to share :)